CORDOVA_UPGRADE_GUIDE.md

```diff
@@ -0,0 +1,235 @@
+# Upgrade Guide To The New Branch PhoneGap/Cordova SDK
+
+In the effort of making the quickly growing functionality of our Web SDK, available to your PhoneGap or Cordova apps, we've merged the two together. This will require some very simple changes to the implementation, but you'll gain a number of very useful functions and the Branch promise of continuous updates and improvements!
```

Comment:
Wow, so this is impressively more difficult than you'd think: putting a code snippet inside a table w/ markdown. Going to revisit after everything below.